CHANGES

```diff
@@ -1,6 +1,19 @@
 This file describes the most significant changes. For more detail, use
 'git log' on a clone of the charm repository.
 
+================================================================================
+What's new in Charm++ 6.10.1
+================================================================================
+
+This is a minor bug-fix release, with the following changes:
+
+- Fix verbs on new InfiniBand hardware/drivers. This fixes some of the observed
+  failures on InfiniBand machines, but not all of them; it is still a known
```

Comment:
I would describe this as simply "Fix verbs layer send completion errors on recent InfiniBand hardware/drivers."